xla/pjrt/gpu/BUILD

```diff
@@ -309,10 +309,8 @@ xla_cc_test(
     name = "se_gpu_pjrt_compiler_aot_test",
     srcs = if_gpu_is_configured(["se_gpu_pjrt_compiler_aot_test.cc"]),
     tags = [
-        "config-cuda-only",
         "gpu",
         "no_oss",
-        "requires-gpu-nvidia",
```

Comment:
We need to keep this tag here because it has some internal meaning that ensures this test runs with a GPU available

Reply:
done